docs/sql/functions/numeric.md

```diff
@@ -12,17 +12,23 @@ The table below shows the available mathematical operators for numeric types.
 | `+` | addition | `2 + 3` | 5 |
 | `-` | subtraction | `2 - 3` | -1 |
 | `*` | multiplication | `2 * 3` | 6 |
-| `/` | division | `4 / 2` | 2 |
+| `/` | float division | `5 / 2` | 2.5 |
+| `//` | division | `5 // 2` | 2 |
 | `%` | modulo (remainder) | `5 % 4` | 1 |
 | `**` | exponent | `3 ** 4` | 81 |
 | `^` | exponent (alias for `**`) | `3 ^ 4` | 81 |
 | `&` | bitwise AND | `91 & 15` | 11 |
 | `|` | bitwise OR | `32 | 3` | 35 |
 | `<<` | bitwise shift left | `1 << 4` | 16 |
 | `>>` | bitwise shift right | `8 >> 2` | 2 |
+| `~` | bitwise negation | `~15` | -16 |
 | `!` | factorial of x. Computes the product of the current integer and all integers below it  | `4!` | 24 |
 
-The modulo, bitwise, and factorial operators work only on integral data types,
+There are two division operators: `/` and `//`.<br/>
```

Comment:
What's up with the line breaks here? Are they really needed?

Reply:
Looks to me that the logic is like:
* whenever there are 2 consecutive newlines: split in different <p>the text</p> elements
* otherwise discard newlines and condensate in a single paragraph (with NO newlines, just a single blob).

Example: https://duckdb.org/docs/sql/window_functions#framing

<br/> is horrible, but also the blob without spaces looks strange. It's your call.

Reply:
Feel free to break a particularly long paragraph, but I don't think we need to break on every sentence. And if you want a break, just use an empty line instead